docs/source/guides/gettingstarted.rst

```diff
@@ -71,6 +71,294 @@ An example for this might be:
 
 ``"Path"`` indicates where the script is, ``"RunOn"`` is the Squirrel VM context (see :doc:`../native/sqvm`) as an expression, and ``"ClientCallback"`` and ``"ServerCallback"`` specify a function call that can be ``"Before"`` and/or ``"After"`` map-spawn.
 
+
+Detailed ``mod.json`` architecture
+==================================
```

Comment:
This uses the wrong thingy, should either be `-` instead of `=`, or be moved into a separate file

Reply:
Fixed with 9bb2c83171b8fce20618064ed5b5475f7a8ec70e